extsort_test.go

```diff
@@ -163,38 +163,46 @@ var _ = Describe("Sorter", func() {
 		Expect(keys(reverse)).To(Equal([]string{"foo", "dau", "baz", "bar"}))
 	})
 
-	It("supports compression", func() {
-		compressed := extsort.New(&extsort.Options{
-			BufferSize:  1024 * 1024,
-			Dedupe:      bytes.Equal,
-			WorkDir:     workDir,
-			Compression: extsort.CompressionGzip,
-		})
-		defer compressed.Close()
-
-		for i := 0; i < 100; i++ {
-			Expect(compressed.Append([]byte("foo"))).To(Succeed())
-			Expect(compressed.Append([]byte("bar"))).To(Succeed())
-			Expect(compressed.Append([]byte("baz"))).To(Succeed())
-			Expect(compressed.Append([]byte("dau"))).To(Succeed())
+	Context("supports compression", func() {
```

Comment:
I would recommend going back to the std test pkg. May I ask why you prefer this?

Reply:
I don't anymore, I wouldn't mind going back to stdlib testing